languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8788,18 +8788,30 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
-      <rule id= "CA_VA_MAL_SE_FINIR" name="ça va mal se finir" default="temp_off">
-          <pattern>
-              <token regexp="yes">ça</token>
-              <token regexp="yes">va</token>
-              <token regexp="yes">mal</token>
-              <marker><token regexp="yes">se</token></marker>
-              <token regexp="yes">finir</token>
-          </pattern>
-          <message>«se» est une expression fautive.</message>
-          <suggestion><match no="4" regexp_match="se" regexp_replace=""/></suggestion>
-          <example correction="">Je pense que ça va mal <marker>se</marker> finir.</example>
-      </rule>
+      <rulegroup id= "CA_VA_MAL_SE_FINIR" name="ça va mal se finir" type="style" default="temp_off">
+          <rule>
+              <pattern>
+                  <token regexp="yes">ça</token>
+                  <token regexp="yes">va</token>
+                  <token regexp="yes">mal</token>
+                  <marker><token regexp="yes">se</token></marker>
+                  <token regexp="yes">finir</token>
+              </pattern>
+              <message>Dans cette expression française familière,« se finir » est une erreur grammaticale. Nous vous suggérons  d’écrire « ça va mal <suggestion><match no="4" regexp_match="se" regexp_replace=""/></suggestion> finir ». </message>
+              <example correction="">Je pense que ça va mal <marker>se</marker> finir.</example>
```

Comment:
The `<suggestion>` element inside `message` will add quotation marks to the message seen by the user, which are not desired in this case. Put the suggestion outside the message. 
Replacing a word with nothing is not intuitive for the user. Better replace `se finir` -> `finir`.

Reply:
Pourrais-tu mettre 'se finir' entre guillemets ?

Reply:
> Replacing a word with nothing is not intuitive for the user. Better replace "se finir" -> "finir".

I meant this: 
```
<marker>
   <token>se</token>
   <token regexp="yes">finir|terminer</token>
</marker>
...
<suggestion><match no="6"/></suggestion>
<example correction="finir">Je pense que ça va mal <marker>se finir</marker>.</example>
```